client/src/components/common/Footer/index.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faEnvelope,
+  faMapMarker,
+  faPhone,
+} from '@fortawesome/free-solid-svg-icons';
+import {
+  faTwitter,
+  faInstagram,
+  faFacebookSquare,
+  faBehance,
+  faSkype,
+  faPinterest,
+} from '@fortawesome/free-brands-svg-icons';
+import logo from '../../../assets/images/berwaz.png';
+import './style.css';
+
+const Footer = () => {
+  return (
+    <div className="main-container">
+      <div className="footer-container">
+        <div className="follow-container">
+          <p>تابعنا على</p>
+          <div className="follow-icons">
+            <a
+              href="https://twitter.com"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <FontAwesomeIcon icon={faTwitter} color="#636D73" size="2x" />
+            </a>
+            <a
+              href="https://pinterest.com"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <FontAwesomeIcon icon={faPinterest} color="#636D73" size="2x" />
+            </a>
+            <a
+              href="https://instagram.com"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <FontAwesomeIcon icon={faInstagram} color="#636D73" size="2x" />
+            </a>
+            <a
+              href="https://www.facebook.com"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <FontAwesomeIcon
+                icon={faFacebookSquare}
+                color="#636D73"
+                size="2x"
+              />
+            </a>
+            <a
+              href="https://www.behance.net"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <FontAwesomeIcon icon={faBehance} color="#636D73" size="2x" />
+            </a>
+          </div>
+        </div>
+        <div className="contact-container">
+          <p className="contact-title">تواصل معنا</p>
+          <div className="contact-icons">
+            <div className="icon-1">
+              <p>غزة - فلسطين</p>
+              <FontAwesomeIcon icon={faMapMarker} color="#636D73" size="lg" />
+            </div>
+            <div className="icon-2">
+              <p>+972-000000</p>
+              <FontAwesomeIcon icon={faPhone} color="#636D73" size="lg" />
+            </div>
+            <div className="icon-3">
+              <p>berwaz@gmail.com</p>
```

Comment:
I made an email for our app, you can start use it
```suggestion
              <p>berwaz.g@gmail.com</p>
```